### ISSUE TITLE
Game stopped loading

### ISSUE THREAD

**Issue by daskew87** [OpenRA/OpenRA#11810]
I had a battle with a friend. Then I closed the game, went out for a walk. Came back and wanted to play him again. Game stopped loading. Restarted computer again and again. Still the same. Reinstalled the game twice. Same problem.

Operating System: Windows (Microsoft Windows NT 6.2.9200.0)
Runtime Version: .NET CLR 4.0.30319.42000
Exception of type `System.IO.FileNotFoundException`: The system cannot find the file specified. (Exception from HRESULT: 0x80070002)
   at System.Reflection.RuntimeAssembly.nLoadFile(String path, Evidence evidence)
   at System.Reflection.Assembly.LoadFile(String path)
   at OpenRA.Sound..ctor(String engineName)
   at OpenRA.Game.Initialize(Arguments args)
   at OpenRA.Program.Run(String[] args)
   at OpenRA.Program.Main(String[] args)

I am running windows 10 home.

Please help me, I love this game. Thankyou very much.

**Comment by Phrohdoh** [comment]
Could you share the contents of your `exception.log`?

**Comment by daskew87** [comment]
Operating System: Windows (Microsoft Windows NT 6.2.9200.0)
Runtime Version: .NET CLR 4.0.30319.42000
Exception of type `System.IO.FileNotFoundException`: The system cannot find the file specified. (Exception from HRESULT: 0x80070002)
   at System.Reflection.RuntimeAssembly.nLoadFile(String path, Evidence evidence)
   at System.Reflection.Assembly.LoadFile(String path)
   at OpenRA.Sound..ctor(String engineName)
   at OpenRA.Game.Initialize(Arguments args)
   at OpenRA.Program.Run(String[] args)
   at OpenRA.Program.Main(String[] args)

**Comment by GraionDilach** [comment]
Duplicate of #11252.

For solution, please go to My Documents/OpenRA and delete the (Sound: lines from) settings.yaml there. And please don't select Output Disabled in the sound settings for now.

**Comment by Phrohdoh** [comment]
Oh I see, it is bailing before we can even get to the mod.
I was of no help here then, sorry.

@GraionDilach close as duplicate?

**Comment by GraionDilach** [comment]
I'd await until the confirmation beforehand though.

**Comment by daskew87** [comment]
Thankyou, I am not a computer wiz, so out of this what do I delete?

Player:
	Name: Dask
	Color: 5EF582

Game:
	Mod: ra
	ViewportEdgeScrollStep: 29.652174

Sound:
	Engine: Null

Graphics:

Server:
	AllowPortForward: False
	Map: 47b40b5c36c10af22122a9255d9963652399326f

Debug:
	SystemInformationVersionPrompt: 1
	UUID: f917acd4-727f-4787-a187-1d2705553273

Keys:

Chat:
	Nickname: Dask

**Comment by GraionDilach** [comment]
```
Sound:
	Engine: Null
```
This block.

**Comment by daskew87** [comment]
You guys are the best. This was brillaint, I went from despair to being happy.

Confirmed.

**Comment by GraionDilach** [comment]
You accidentally reopened this.
